tools/buildgen/_generate_projects.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2015 gRPC authors.
```

Comment:
Nit: `git mv` so we can review just the portions of the file that have actually changed.

Reply:
I can't find a good way to redo the move and let GitHub realize that these two files should share the same history. So, I changed the name back.